rlclientlib/logger/async_batcher.h

```diff
@@ -153,9 +162,39 @@ namespace reinforcement_learning { namespace logger {
 
     RETURN_IF_FAIL(collection_serializer.finalize(status));
 
+    if (_events_counter_status == events_counter_status::ENABLE) {
+      unsigned int _buffer_start_event_number;
```

Comment:
Can you remove _ in the variable name? A local variable should not have _ prefix based on the pattern we have.